errbot/backends/xmpp.py

```diff
@@ -31,6 +32,9 @@
     """)
     sys.exit(-1)
 
+# LRU to cache the JID queries.
+IDENTIFIERS_LRU = 1000
```

Comment:
According to the documentation for `lru_cache`, it performs better when using a power of 2, so I suggest changing this to `1024`